safor/index.html

```diff
@@ -79,29 +79,29 @@ <h1>WHY SAFOR?</h1>
                     <h2>FOR PEOPLE</h2>
             </div>
             <ul>
-                <li><img src="./images/section5/pentagon.svg" alt="pentagon"></li>
-                <li>Do not waste time to explain your location and nature of the threat, guard patrols get <br id="get-br">your alarm in 1 second and advances to you no matter whether you are in motion or not.</li>
-                <li><img src="./images/section5/star.svg" alt="star"></li>
-                <li>You will be foundregardless of whether you are in town or on a trip. <br id="trip-br">The service covers the entire territory of the country and allows you to specify <br id="specify-br">the exact location even when in a crowded place</li>
-                <li><img src="./images/section5/triangle.svg" alt="triangle"></li>
-                <li>And we will help in any situation. Only after coming to you the calling <br id="calling-br">will be closed by the patrol arrived, whose goal is to protect and help you.</li>
+                <li><img src="images/section5/pentagon.svg" alt="pentagon"></li>
```

Comment:
Image and text for this image - one item. Show nesting in tags, code will be more readable. Example :
```
<li>
 <img/>
 <p></p>
</li>
```